lib/ASN1/Universal/Integer.php

```diff
@@ -60,14 +65,13 @@ protected function getEncodedValue()
         $contentLength = $this->getContentLength();
 
         if ($numericValue < 0) {
```

Comment:
It appears setting saving the result of gmp_init() in the constructor causes problems elsewhere in the tests. I'm using PHP5.5 at the moment, and since this has gmp resources, doing assertEquals fails. 

I might avoid this for now to keep this change simple.